cli/bpmetadata/tfconfig.go

```diff
@@ -235,13 +244,38 @@ func getBlueprintInterfaces(configPath string) (*BlueprintInterface, error) {
 		variables = append(variables, v)
 	}
 
+	p := hclparse.NewParser()
+	variableFile, hclDiags := p.ParseHCLFile(filepath.Join(configPath, "variables.tf"))
+	if hclDiags.HasErrors() {
+		Log.Info("Failed to parse HCL: ", "diags: ", hclDiags)
+	}
+	variableContent, _, hclDiags := variableFile.Body.PartialContent(variableSchema)
+	if hclDiags.HasErrors() {
+		Log.Info("Failed to parse variable content: %v", hclDiags)
+	}
+	variableOrderKeys := make(map[string]int)
+	for i, block := range variableContent.Blocks {
+		// We only care about variable blocks.
+		if block.Type != "variable" {
+			continue
+		}
+		// We expect a single label which is the variable name.
+		if len(block.Labels) != 1 {
+			Log.Info("Variable block has no name: %v", block)
+		}
+		variableOrderKeys[block.Labels[0]] = i
```

Comment:
Could length of block.Labels be 0? It would be an issue with block.Labels[0] access.

Reply:
Above lines checked the `block.Labels` length, let me add `continue` for empty `Lables`.

Reply:
Done